pkg/agent/catalog/workloadattestor.go

```diff
@@ -28,7 +29,7 @@ func (repo *workloadAttestorRepository) Versions() []catalog.Version {
 func (repo *workloadAttestorRepository) BuiltIns() []catalog.BuiltIn {
 	return []catalog.BuiltIn{
 		docker.BuiltIn(),
-		k8s.BuiltIn(),
+		// k8s.BuiltIn(),
```

Comment:
Why this change?

Reply:
we are not using the k8s plugin because it wont be applicable to our use-case. And this plugin is making use of some older vulnerable packages thereby introducing some of the vulnerabilities that have been reported.